tests/integration/test_active_members.py

```diff
@@ -0,0 +1,216 @@
+import numpy as np
+from analyzer.assess_engagement import assess_engagement
+from analyzer.utils.activity import Activity
+
+
+def test_no_active():
+    acc_names = []
+    acc_count = 5
+    for i in range(5):
+        acc_names.append(f"user{i}")
+
+    acc_names = np.array(acc_names)
+
+    int_mat = {
+        Activity.Reply: np.zeros((acc_count, acc_count)),
+        Activity.Mention: np.zeros((acc_count, acc_count)),
+        Activity.Reaction: np.zeros((acc_count, acc_count)),
+    }
+
+    activity_dict = {
+        "all_joined": {},
+        "all_joined_day": {"0": set()},
+        "all_consistent": {},
+        "all_vital": {},
+        "all_active": {},
+        "all_connected": {},
+        "all_paused": {},
+        "all_new_disengaged": {},
+        "all_disengaged": {},
+        "all_unpaused": {},
+        "all_returned": {},
+        "all_new_active": {},
+        "all_still_active": {},
+        "all_dropped": {},
+        "all_disengaged_were_newly_active": {},
+        "all_disengaged_were_consistently_active": {},
+        "all_disengaged_were_vital": {},
+        "all_lurker": {},
+        "all_about_to_disengage": {},
+        "all_disengaged_in_past": {},
+    }
+
+    WINDOW_D = 7
+
+    # window index
+    w_i = 0
+
+    INT_THR = 1  # minimum number of interactions to be active
+    UW_DEG_THR = 1  # minimum number of accounts interacted with to be active
+    PAUSED_T_THR = 1  # time period to remain paused
+    CON_T_THR = 4  # time period to be consistent active
+    CON_O_THR = 3  # time period to be consistent active
+    EDGE_STR_THR = 5  # minimum number of interactions for connected
+    UW_THR_DEG_THR = 5  # minimum number of accounts for connected
+    VITAL_T_THR = 4  # time period to assess for vital
+    VITAL_O_THR = 3  # times to be connected within VITAL_T_THR to be vital
+    STILL_T_THR = 2  # time period to assess for still active
+    STILL_O_THR = 2  # times to be active within STILL_T_THR to be still active
+    DROP_H_THR = 2
```

Comment:
description for DROP_H_THR: # time periods into the past (history) to be newly active for computing dropped
description for DROP_I_THR: # consecutive time periods into the past to have not been active for computing dropped